.changeset/serious-coats-stare.md

```diff
@@ -0,0 +1,12 @@
+---
+"@digdir/designsystemet": patch
+---
+
+fix: dark mode color adjustments
+- Reduced the saturation of the colors in dark mode by about 50%.
+- Made the background- and surface colors a bit darker.
+- Made the text-default color a bit brighter for better contrast.
+
+fix: general color adjustments
+- Changed the luminance of border-default to work on top of surface-hover with 3:1 contrast.
+- Changed the luminance of text-subtle and border-strong to work on top of surface-hover with 4.5:1 contrast.
```

Comment:
Suggest making these two separate changesets, and remove "fix:".

Wo probably need a changeset for the theme package as well, since the colors the have changed. Could probably just point the consumer to the changelog for `@digdir/designsystemet`